### PR TITLE
Add instructions to template/ios/Podfile for enabling hermes

### PR DESCRIPTION
**Pull request by SConaway**
## Summary

Just thought I'd add these instructions so devs don't have to check the docs. Also, it makes iOS match Android with instructions in the configuration files
## Changelog
N/A (in my opinion)

## Test Plan
N/A (because not a code change)

### CODE REVIEW

File: template/ios/Podfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ target 'HelloWorld' do
   config = use_native_modules!
 
   use_react_native!(:path => config[:reactNativePath])
+  # Or, to enable hermes on iOS, comment out the above line and uncomment the below:
+  # use_react_native!(:path => config[:reactNativePath], :hermes_enabled => true)
```
Comment (alloy) [inline template/ios/Podfile:11]: What do you think about doing it like so?

```ruby
use_react_native(
  :path => config[:reactNativePath],
  # Or, to enable hermes on iOS, uncomment the below line:
  # :hermes_enabled => true
)
```

**Review by alloy — COMMENTED** [review]
Thanks a bunch for taking the time! 🙏 

This is already great as is, but wondering if it feels more natural to not have to comment a whole duplicated line and if this better sets us up for other potential future options.

File: template/ios/Podfile (hunk 1 above)
Comment (SConaway) [reply template/ios/Podfile:11]: I like that idea better! I just didn't think about it.

**Comment by SConaway** [comment]
Done!

**Comment by TheSavior** [comment]
Should the template just have `:hermes_enabled => false`? That should also help make it clear how to enable it even for people who don't enable it initially

**Comment by alloy** [comment]
@TheSavior Smart, yeah that’s a good call 👍

**Review by kelset — APPROVED** [review]
LGTM

**Review by alloy — APPROVED** [review]
Thanks!

**Pull request closed without merging**